src/daria/analysis/segmentationcomparison.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+import numpy as np
+
+
+def compare_segmentations(*segmentations, **kwargs) -> np.ndarray:
+    """
+    Comparison of segmentations.
+
+    Args:
+        segmentations (asterisk argument): Allows to provide an arbitraty
+                number of segmented images to be compared
+        optional keyword arguments (kwargs):
+            components (list): list of the different components that are
+                found in the segmented images. So far only two are allowed
+                to be provided.
+            gray_colors (np.ndarray): array of three different scales of
+                gray, one for each of the different combinations of
+                components in the segmentations.
```

Comment:
Mention that these have to be provided in RGB space (and not in grayscale space), as long it is implemented as right now.

Reply:
I actually thought about this, but aparently I forgot to add it myself. Thanks!